src/Uno.Extensions.Reactive/Collections/Facades/Composite/CompositeEnumerator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Uno.Extensions.Collections.Facades.Composite;
+
+
+// TODO: Uno : use enumerators!
```

Comment:
New item in the backlog?

Reply:
On going, done in my current working branch